scripts/setup/terminate-psql-sessions

```diff
@@ -26,11 +26,16 @@ root)
     ;;
 esac
 
+if [[ -n "${PGDATABASE}" ]] && [[ -n "${PGHOST}" ]] && [[ -n "${PGUSER}" ]]; then
+    psql=(psql)
+fi
```

Comment:
What's the purpose of this block?

Reply:
In the current version this script is run from `restore-backup` command as root. So psql command would be called as 

```
su -s /usr/bin/env - -- "$POSTGRES_USER" psql postgres "$POSTGRES_USER"
```

This wont work in docker since there is no `postgres` user in docker zulip.

I thought an easier way to handle the case of docker is to check whether the environment variables `PGDATABASE`, `PGHOST` and `PGUSER` is set. If they are set envoking the `psql` command with the query is enough. No need to pass any other parameters to the psql command.